### PR TITLE
ddl: improve index change test.

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: ddl/index_change_test.go [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+func (s *testIndexChangeSuite) SetUpSuite(c *C) {
+	s.store = testCreateStore(c, "test_index_change")
+	s.dbInfo = &model.DBInfo{
+		Name: model.NewCIStr("test_index_change"),
+		ID:   1,
+	}
+	err := kv.RunInNewTxn(s.store, true, func(txn kv.Transaction) error {
+		t := meta.NewMeta(txn)
+		err1 := errors.Trace(t.CreateDatabase(s.dbInfo))
```
Comment (zimulala) [inline ddl/index_change_test.go]: Return the error directly.

File: ddl/index_change_test.go [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+		prevState = job.SchemaState
+		var err error
+		switch job.SchemaState {
+		case model.StateDeleteOnly:
+			deleteOnlyTable, err = getCurrentTable(d, s.dbInfo.ID, tblInfo.ID)
+			if err != nil {
+				checkErr = errors.Trace(err)
+			}
+		case model.StateWriteOnly:
+
```
Comment (zimulala) [inline ddl/index_change_test.go]: Remove this empty line.

File: ddl/index_change_test.go [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+		return errors.Trace(err)
+	}
+	idx := tbl.Indices()[0]
+	doesExist, _, err := idx.Exist(txn, types.MakeDatums(indexValue), handle)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if exists != doesExist {
+		if exists {
+			return errors.Errorf("index should exists")
```
Comment (zimulala) [inline ddl/index_change_test.go]: `New` is enough?

**Comment by zimulala** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**